migrations/v6-MIGRATION.md

```diff
@@ -26,6 +26,7 @@
 | `UpgradeInfo`           | `PurchaseParams.Builder.oldProductId` and `PurchaseParams.Builder.googleProrationMode` |
 | `ProductChangeCallback` | `PurchaseCallback`                                                                     |
 
+`Offering`, `Offerings`, `Package`, and `StoreProduct` are no longer Parcelable.
```

Comment:
👍 I do wonder if this should go to a different section in this doc... Maybe `Breaking changes`?  I think this is ok thoug

Reply:
josh and i are planning a whole revamp of this doc! stay tuned :)